src/main/java/com/ericsson/eiffel/remrem/semantics/schemas/SchemaFile.java

```diff
@@ -24,42 +25,42 @@
  *
  */
 public class SchemaFile {
-	private static String fileContent;
-	private final static JsonParser parser = new JsonParser();
-	private static int count = 0;
-	static boolean isItemObject = false;
-	private static String oldName = "";
-	private static String fileName;
-	private static boolean isMeta = false;
-	private static boolean isEnumType = false;
-	private ArrayList<String> allJsonFileNames;
+	private JsonParser parser = new JsonParser();
+	private boolean isEvent;
+	private String eventName;
```

Comment:
As the code is written I suggest that this is not an instance variable but handled by passing values in the methods.